dockerfiles/init/modules/keycloak/files/che/login/messages/messages_en.properties

```diff
@@ -0,0 +1,240 @@
+product=Eclipse Che
```

Comment:
how to we keep in sync this file with new keycloack releases ?
what are the changes with upstream project ?

should it be better to apply patches ? so we can track changes easily